crates/ironrdp-web/src/session.rs

```diff
@@ -159,27 +158,27 @@ impl SessionBuilder {
         self.clone()
     }
 
-    /// Required
-    pub fn hide_pointer_callback(&self, callback: js_sys::Function) -> SessionBuilder {
-        self.0.borrow_mut().hide_pointer_callback = Some(callback);
-        self.clone()
-    }
-
-    /// Required
-    pub fn hide_pointer_callback_context(&self, context: JsValue) -> SessionBuilder {
-        self.0.borrow_mut().hide_pointer_callback_context = Some(context);
+    /// Required.
+    ///
+    /// #### Callback signature:
```

Comment:
Nitpick: I think a single `#` is enough for sections in doc comments

Reply:
I don't like it, its huge 😆 
![image](https://github.com/Devolutions/IronRDP/assets/3994505/f14f83d6-6df4-442c-9980-f3331082be03)